internal/proxy/impl.go

```diff
@@ -1653,7 +1738,16 @@ func (node *Proxy) CalcDistance(ctx context.Context, request *milvuspb.CalcDista
 		}, nil
 	}
 
-	if vectorsLeft.Dim == vectorsRight.Dim && vectorsLeft.GetFloatVector() != nil && vectorsRight.GetFloatVector() != nil {
```

Comment:
do we need to check left and right vector has same type here?
Either both floatVectors or both binaryVectors